release/artifacts.yaml

```diff
@@ -128,7 +128,7 @@
   type: jar-scala
 - target: //libs-scala/auth-utils:auth-utils
   type: jar-scala
-- target: //libs-scala/caching:caching
+- target: //ledger/caching:caching
```

Comment:
Can you re-sort these?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6043/commits/5e0e4b683bb21748043e75e9fb1e25adf1f3e30e